pkg/oci/utils.go

```diff
@@ -302,12 +303,25 @@ func vmConfig(ocispec CompatOCISpec, config RuntimeConfig) (vc.Resources, error)
 	}
 
 	// round up memory to 1MB
-	mem := uint((memBytes + (1024*1024 - 1)) / (1024 * 1024))
+	resources.Memory = uint((memBytes + (1024*1024 - 1)) / (1024 * 1024))
+
+	if ocispec.Linux.Resources.CPU == nil ||
+		ocispec.Linux.Resources.CPU.Quota == nil ||
+		ocispec.Linux.Resources.CPU.Period == nil {
+		return resources, nil
+	}
+
+	quota := *ocispec.Linux.Resources.CPU.Quota
+	period := *ocispec.Linux.Resources.CPU.Period
+
+	if quota < 0 {
```

Comment:
We want to test:
```go
if quota <= 0 {
```
as we did for the memory. Otherwise, in case `quota == 0`, we are gonna end up with `resources.VCPUs = 0` because `(period -1) / period` will return 0.